internal/io/file/file_sink.go

```diff
@@ -102,6 +103,15 @@ func (m *fileSink) Configure(props map[string]interface{}) error {
 			c.Delimiter = ","
 		}
 	}
+	if c.Compress != ZLIB && c.Compress != GZIP && c.Compress != FLATE && c.Compress != NONE_COMPRESS && c.Compress != "" {
+		return fmt.Errorf("compressAlgorithm must be one of none, zlib, gzip or flate")
+	}
```

Comment:
Can we maintained the support compress algorithm type in list, so that if we support the new compress algorithm, we don't have to update here.